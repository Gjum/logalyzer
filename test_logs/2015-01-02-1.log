[04:00:00] [Server thread/INFO]: [Server] leave HHL
[04:00:00] [Server thread/INFO]: HHL lost connection: TextComponent{text='Disconnected', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}
[04:00:00] [Server thread/INFO]: §6HHL§r left the game
[05:00:00] [Server thread/INFO]: [Server] rejoin Ulexos
[05:00:00] [User Authenticator #8/INFO]: UUID of player Ulexos is a1d94087-e9c0-412f-b90f-cff2c224d44f
[05:00:00] [Server thread/INFO]: Ulexos lost connection: TextComponent{text='You logged in from another location', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}
[05:00:00] [Server thread/INFO]: §2Ulexos§r left the game
[05:00:00] [Server thread/WARN]: handleDisconnection() called twice
[05:00:00] [Server thread/INFO]: Ulexos[/95.91.242.240:54254] logged in with entity id 517154 at (-6382.5, 44.0, -957.5)
[05:00:00] [Server thread/INFO]: §2Ulexos§r joined the game
[06:00:00] [Server thread/INFO]: [Server] midnight-wrap, still on: Ulexos, Offlinegott
[06:00:00] [Server thread/INFO]: [Server] Session HHL 05+24 - 04 82800s
[06:00:00] [Server thread/INFO]: [Server] Online  Ulexos 07+24 -
[06:00:00] [Server thread/INFO]: [Server] Online  Offlinegott 08+24 -