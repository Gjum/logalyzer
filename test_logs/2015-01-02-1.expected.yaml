first_event: 1420167600
last_event: 1420174800
online:
  Offlinegott: [9f352de0-b533-425c-9435-89b918ae4602, 1420095600, 1]
  Ulexos: [a1d94087-e9c0-412f-b90f-cff2c224d44f, 1420171200, 1]
started: false
stopped: null
times:
- [808e727f-895a-4ac2-b246-4b6da2ca9451, 1420084800, 1420167600, HHL]
- [a1d94087-e9c0-412f-b90f-cff2c224d44f, 1420092000, 1420171200, Ulexos]
